src/main/java/org/broadinstitute/consent/http/models/support/SupportRequestComment.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.consent.http.models.support;
+
+/**
+ * Represents comment describing issue for a Zendesk support ticket https://broadinstitute.zendesk.com/api/v2/requests.json
+ */
+public class SupportRequestComment {
+    private String body;
+
+    public SupportRequestComment(String description, String url) {
+        this.body = description + "\n\n------------------\nSubmitted from: " + url;
```

Comment:
optional small nit: maybe description and url should be class fields, then getBody combines them to a string? just in case we need that data separately in the future.

Reply:
ohhh okay, that makes sense. all good!